values.yaml

```diff
@@ -232,6 +228,13 @@ global:
         memory: "50Mi"
         cpu: "20m"
 
+  # The Docker image for envoy to use as the proxy sidecar when performing
+  # Connect injection or using the mesh/terminating/ingress gateways.
```

Comment:
Could we rephrase or break up this sentence? I think it now reads a bit confusing. Maybe something like:

```
imageEnvoy is the name (and tag) of the Envoy Docker image used for the
connect-injected sidecar proxies and the mesh, terminating, and ingress gateways.
```

Reply:
```suggestion
  # imageEnvoy is the name (and tag) of the Envoy Docker image used for the
  # connect-injected sidecar proxies and the mesh, terminating, and ingress gateways.
```